src/Raven.Server/Raven.Server.csproj

```diff
@@ -98,7 +98,7 @@
     <None Update="Properties\Settings\settings_posix.json">
       <CopyToOutputDirectory>Never</CopyToOutputDirectory>
     </None>
-    <None Update="settings.json">
+    <None Update="settings.default.json">
```

Comment:
Did you also update the build scripts?

Reply:
We need to change target path to `settings.default.json` here:
https://github.com/ravendb/ravendb/blob/v4.0/scripts/package.ps1#L81

Please check the above by running: `.\build.ps1 -WinX64`

Nothing to change on the Docker side.